### PR TITLE
Network-log ecl version on init

### PR DESCRIPTION
**Pull request by epa095**
**Pre un-WIP checklist**
- [x] Statoil tests pass locally
- [ ] Have completed graphical integration test steps

**Depends on**
* Statoil/libecl#

### CODE REVIEW

**Comment by epa095** [comment]
Internal tests passed on jenkins

File: python/python/res/job_queue/job_manager.py [hunk 1]
```diff
@@ -122,12 +123,17 @@ def __init__(self, module_file="jobs.py", json_file="jobs.json", error_url=None,
...
         pw_entry = pwd.getpwuid(os.getuid())
         self.user = pw_entry.pw_name
+        v = EclVersion()
         logged_fields= {"status": "init",
                         "python_sys_path": sys.path,
                         "pythonpath": os.environ.get('PYTHONPATH', ''),
+                        "ecl_major_version": v.major,
+                        "ecl_minor_version": v.minor,
+                        "ecl_micro_version": v.micro,
+                        "ecl_version": v.versionString(),
```
Comment (joakim-hove) [inline python/python/res/job_queue/job_manager.py]: I would have thought that `"ecl_version" : v.versionString()` should suffice?

**Review by joakim-hove — DISMISSED** [review]
The `EclVersion` is the least interesting `ResVersion` and `ErtVersion`are in the pipeline; maybe wait for those?

File: python/python/res/job_queue/job_manager.py (hunk 1 above)
Comment (epa095) [reply python/python/res/job_queue/job_manager.py]: Yeah, I considered it, and I am still uncertain. Maybe we would like to have easy access to see who runs e.g. 2.1.$whatever, or how many runs 2.$whatever. As long as the v.versionString() format remains locked it is of course enough for that, but it feels ugly to lock that format. On the other hand, it is a fricking version string, why would we every want to change its format?

**Comment by epa095** [comment]
Ref ResVersion and ErtVersion, are they in now, and/or will they be inn before feature freeze?

**Comment by joakim-hove** [comment]
> Ref ResVersion and ErtVersion, are they in now, and/or will they be inn before feature freeze?

They are very much on their way in. `ResVersion` here: #91

**Comment by epa095** [comment]
Changed it now to send up ecl and res version, and for each of them we only send up a single string like "2.2.git" instead of one field each for major,minor, and micro.

**Comment by epa095** [comment]
Review plz  @joakim-hove

**Comment by joakim-hove** [comment]
Sorry it has been slow. Joakim

**Pull request merged**